node/src/chain_spec.rs

```diff
@@ -116,32 +148,34 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
 		"canvas_local",
 		ChainType::Local,
 		move || {
-			// TODO [#76]: Right now these are being left as default values, but in the old `canvas-node`
-			// we had some non-default authorities, as well as accounts for the faucent and
-			// `ink-waterfall`. We'll need to add those back in
 			testnet_genesis(
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
-				vec![get_from_seed::<AuraId>("Alice"), get_from_seed::<AuraId>("Bob")],
+				local_testnet_root(),
+				local_testnet_authorities(),
 				vec![
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_account_id_from_seed::<sr25519::Public>("Bob"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie"),
-					get_account_id_from_seed::<sr25519::Public>("Dave"),
-					get_account_id_from_seed::<sr25519::Public>("Eve"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+					local_testnet_root(),
+					// AccountId of the Canvas Testnet faucet
+					hex!("18c64aa111a8a0e6e4eed41d6d906c7614d745e48be3cfc13b6128e1d51f4405").into(),
```

Comment:
@lovelaced We want to use the Relaychain token for Canvas, do you know if the other parachains which do this just use the Rococo faucet then? We could remove this then.

Reply:
Token symbol is still set to "CAN" in this spec, though.

Reply:
@athei  I'm working on the currency related changes ;)

Reply:
@cmichi statemint on rococo (which doesn't exist at the moment) did in fact use ROC which you can just use the Rococo faucet for.

Reply:
Okay, I've removed the faucet from the chain-spec!